lib/pages/main/wallet_contents/explorer/explorer_result_page.dart

```diff
@@ -76,21 +81,39 @@ class _ExplorerResultPageState extends State<ExplorerResultPage> {
   }
 
   //Gets info from the backend and stores it in the store
-  void getInfo() {
+  void getInfo() async {
     setState(() {
       isLoading = true;
       error = null;
     });
+
     if (resultType == ExplorerResultType.tick) {
-      qubicLi.getExplorerTickInfo(tick!).then((value) {
+      try {
+        final futures = await Future.wait([
+          qubicArchiveApi.getExplorerTick(tick!),
+          qubicArchiveApi.getExplorerTickTransactions(tick!),
+        ]);
+        tickData = futures[0] as ExplorerTickDto;
+        transactions = futures[1] as List<ExplorerTransactionDto>;
+        final computers = await qubicArchiveApi.getComputors(tickData!.epoch);
+        tickData?.tickLeaderId = computers.identities[tickData!.computorIndex];
+        isLoading = false;
+        setState(() {});
+      } catch (err) {
         setState(() {
-          tickInfo = value;
-          isLoading = false;
+          error = err is AppError ? err.message : err.toString();
         });
-      },
-          onError: (err) => setState(() {
-                error = err.toString().replaceAll("Exception: ", "");
-              }));
+      }
+
+      // qubicLi.getExplorerTickInfo(tick!).then((value) {
```

Comment:
please remove commented code if not needed anymore

Reply:
Done